app/src/components/notifications/NotificationManager.vue

```diff
@@ -21,11 +28,24 @@ const { notifications } = storeToRefs(useNotificationStore());
                 leave-to-class="opacity-0"
             >
                 <LNotification
-                    v-for="notification in notifications"
+                    v-for="notification in notifications.filter((c) => c.type == 'toast')"
                     :key="notification.id"
                     :notification
                 />
             </TransitionGroup>
         </div>
     </div>
+    <div
+        aria-live="assertive"
+        class="inset-x-0 flex items-end sm:items-start"
+        v-if="type == 'banner'"
+    >
+        <div class="w-full">
+            <LNotification
+                v-for="notification in notifications.filter((c) => c.type == 'banner')"
+                :key="notification.id"
+                :notification
+            />
+        </div>
+    </div>
```

Comment:
I think this should rather belong in it's own component. The thought behind the `NotificationManager` before was that you render it once. Now it can be a bit confusing, because it matters where you render it. A `NotificationManager` with `type=banner` should not be rendered at the end of the `body` tag, where a `toast` should. I think it would be clearer to split it in something like a `NotificationToastManager` and `NotificationBannerManager`.